core/src/BitMatrix.h

```diff
@@ -206,4 +214,26 @@ class BitMatrix
 	}
 };
 
+/**
+ * @brief Inflate scales a BitMatrix up and adds a quite Zone plus padding
+ * @param matrix input to be expanded
+ * @param width new width in bits (pixel)
+ * @param height new height in bits (pixel)
+ * @param quietZone size of quite zone to add in modules
+ * @return expanded BitMatrix, maybe move(input) if size did not change
+ */
+BitMatrix Inflate(BitMatrix&& input, int width, int height, int quietZone);
```

Comment:
We may want to use `const BitMatrix& input` instead for the sake of coherence (see Deflate)? Accept only rvalue makes it look like the method is written to be called from particular site.

Reply:
My reasoning was performance: in case the requested size of the matrix is less than that of the input (like the 'I don't care'-value 0) the input can simply be passed on. If that would be a `const &` then it would need to be copied no matter what.

The performance impact is admittedly completely unnoticeable, so it was merely a 'do the right thing' kind of thinking.